content/foundations/color.mdx

```diff
@@ -236,6 +237,49 @@ Not all colors pair well with each other. There are combinations of backgrounds
   src="https://user-images.githubusercontent.com/378023/156498652-b1121c86-a670-4a92-9a8d-9db3c7e0ed9a.png"
 />
 
+## Accessibility
+When designing with colors consider the following to make your UI visually accessible.
+
+### Assure adequate color contrast
+Color contrast between text and its background must meet [required WCAG standards](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html). 
+
+The contrast requirements are:
+- 4.5:1 for normal text
+- 3:1 for large text (>24px)
+- 3:1 for UI elements and graphics
+- decorational and disabled elements have no contrast requirement
+
+Use an [online contrast checker](https://webaim.org/resources/contrastchecker/) or a [figma plugin](https://primer.style/design/accessibility/tools#contrast-plugin) to test your contrast. 
+
+If you work with the [functional colors](#color-roles-recipes), this work has already been done for you. We checked all recommended combinations, so you are all set.
+
+### Don't rely on color alone
+<Box mb={3} display="flex" alignItems="flex-start" alignItems="center" flexDirection={["column", "column", "column", "column", "row"]} sx={{gap: 3}}>
+  <Box as="p" mt="0">
+    <Heading sx={{mt: 0, mb: 3, fontSize: 2}} as="h4">Show state with more than color</Heading>
+    Color vision deficiency is different for different people. To make sure everyone can understand and use your UI you should show state with more than a change in color.
+For example by using icons or changing the content.
+  </Box>
+  <img
+    width="456"
+    alt="Icon supporting color to show error"
+    src="https://user-images.githubusercontent.com/813754/187186864-8a825434-978b-4e21-bac2-9c1235d75e2c.png"
+  />
+</Box>
+
+<Box mb={3} display="flex" alignItems="flex-start" alignItems="start" flexDirection={["column", "column", "column", "column", "row"]} sx={{gap: 3}}>
+  <Box as="p" mt="0">
+      <Heading sx={{mt: 0, mb: 3, fontSize: 2}} as="h4">Connect labels to graphs with lines or patterns</Heading>
+      For charts and graphs you can position the labels on top or close to each section.
+      You can also use patterns to distinguish different parts.
+  </Box>
+  <img
+      width="456"
+      alt="Graph with accessible legend"
```

Comment:
```suggestion
      alt="Two graphs. One is a line graph and one is a pie graph. The line graph is demonstrating using unique shapes for each comparison line's plot symbol. The pie chart is demonstrating how each slice's label can use a line to point to its corresponding slide."
```

Adding a little more detail here about how different distinguishing techniques can be utilized.